web/pdf_find_controller.js

```diff
@@ -281,26 +283,26 @@ function normalize(text) {
       }
 
       if (p5) {
-        // Compound word with a line break after the hyphen.
-        // Since the \n isn't in the original text, o = 3 and n = 3.
-        shiftOrigin += 1;
-        eol += 1;
-        return p5.replace("\n", "");
-      }
-
-      if (p6) {
-        // "X-\n" is removed because an hyphen at the end of a line
-        // with not a space before is likely here to mark a break
-        // in a word.
+        // In "X-\ny", +-\n" is removed because an hyphen at the end of a line
```

Comment:
Nit: A missing quote here?
```suggestion
        // In "X-\ny", "-\n" is removed because an hyphen at the end of a line
```